monai/bundle/reference_resolver.py

```diff
@@ -172,6 +174,39 @@ def get_resolved_content(self, id: str, **kwargs):
         """
         return self._resolve_one_item(id=id, **kwargs)
 
+    @classmethod
+    def resolve_relative_ids(cls, id: str, value: str) -> str:
```

Comment:
It's an internal function but I think we should have more description of what this is actually doing here. The meaning of the relative IDs "@#A" vs "@##A" or something else isn't explained, it would help to motivate what this does here. The example doesn't demonstrate calling the function with inputs and expected outputs.

Reply:
Hi @ericspod ,

Sure, added more description in the latest commit.

Thanks for your suggestion.